docs/self-managed/identity/orchestration-identity/configuration.md

```diff
@@ -0,0 +1,49 @@
+---
+id: configuration
+title: "Configuration"
+description: "Learn about the Identity configuration options available in your Orchestration cluster."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::note
+As a Spring Boot application, Identity supports any standard
```

Comment:
🔧  effectively Identity is part of the Camunda 8 application, I would thus keep this statement general with regards to Camunda 8 as spring boot application.
```suggestion
As a Spring Boot application, Camunda 8 supports any standard
```